corehq/apps/userreports/es/data_source.py

```diff
@@ -44,29 +44,25 @@ def __init__(self, domain, config_or_config_id, filters, aggregation_columns, co
                 )
             self._column_configs[column.column_id] = column
 
-    # @property
-    # def aggregation_columns(self):
-    #     return self._aggregation_columns + [
-    #         deferred_filter.field for deferred_filter in self._deferred_filters.values()
-            # if deferred_filter.field not in self._aggregation_columns]
+    @property
+    def aggregation_columns(self):
+        # TODO add deferred filters to support mobile ucr
+        return self._aggregation_columns
 
     @property
     def config(self):
         if self._config is None:
             self._config, _ = get_datasource_config(self._config_id, self.domain)
         return self._config
 
-    # @property
-    # def engine_id(self):
-    #     return get_engine_id(self.config)
-    #
-    # @property
-    # def column_configs(self):
-    #     return self._column_configs.values()
+    @property
+    def column_configs(self):
+        return self._column_configs.values()
 
     @property
     def table_name(self):
-        return get_table_name(self.domain, self.config.table_id)
+        # TODO make this the same function as the adapter
```

Comment:
not sure it matters but i'm not sure i understand this todo. is the adapter the `IndicatorAdapter`?